Formula/s/sfsexp.rb

```diff
@@ -0,0 +1,60 @@
+class Sfsexp < Formula
+  desc "Small Fast S-Expression Library"
+  homepage "https://github.com/mjsottile/sfsexp"
+  url "https://github.com/mjsottile/sfsexp/releases/download/v1.4.1/sfsexp-1.4.1.tar.gz"
+  sha256 "15e9a18bb0d5c3c5093444a9003471c2d25ab611b4219ef1064f598668723681"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "doxygen" => :build
+  depends_on "libtool" => :build
+  uses_from_macos "m4" => :build
+
+  def install
+    system "./configure", "--disable-silent-rules", *std_configure_args(prefix: prefix)
+    system "make"
+    system "make", "install"
+    system "doxygen"
+    doc.mkpath
+    man3.mkpath
+    libexec.mkpath
+    doc.install Dir["doxygen/html/*"]
+    man3.install "doxygen/man/man3/IO.3" => "sexp_I0.3"
+    man3.install Dir["doxygen/man/man3/{[!IO.3]}*"]
```

Comment:
If make doesn't do this I don't think we should

Reply:
removed (and removed doxygen dependency)